src/components/TopBar/CallTime.vue

```diff
@@ -179,6 +187,10 @@ export default {
 	mounted() {
 		// Start the timer when mounted
 		this.timer = setInterval(this.computeElapsedTime, 1000)
+		this.untilOneHourHint = (1000 * 60 * 60) - this.timer + 1
```

Comment:
If I remember correctly, `this.timer` stores internal id of the interval, to be able to reset it (not the result)
Can we maybe use `this.callTime` here?

Reply:
`this.callTime` is still undefined, it gets its value after `this.computeElapsedTime` which is called in a delay of 1s.

Reply:
this.timer is also can't be used here:
![image](https://github.com/nextcloud/spreed/assets/93392545/53bfbc2c-7208-4543-aa35-8e0c326c7146)
![image](https://github.com/nextcloud/spreed/assets/93392545/0ad8927b-aba3-4c9e-a935-f1636ce784cd)

Reply:
So, we know the time of call start from `conversation.callStartTime`, which is fetched only every 30 seconds
BUT: this value is equal to `message.timestamp` of system message with systemMessage: `call_started`

So either we could patch a conversation in conversationStore with this value, or return back to watcher, and check if `callStart !== 0 && !timeout`, to start it only once